### PR TITLE
Filter sort by

### PR DESCRIPTION
**Pull request by MissMelisa**
Add a new filter to classify countries by language, borders, and areas
image

### CODE REVIEW

File: src/Components/Countries.jsx [hunk 1]
```diff
@@ -34,6 +40,8 @@ Countries.propTypes = {
   currency: PropTypes.string,
   dialCode: PropTypes.number,
   language: PropTypes.string,
+  area: PropTypes.number,
+  borders: PropTypes.number,
```
Comment (falecci) [inline src/Components/Countries.jsx:44]: `borders` should be a `string`, right?

File: src/Components/Countries.jsx [hunk 2]
```diff
@@ -19,10 +22,13 @@ function Countries({
...
-        <span className={styles.currency}>Currency: {currency}</span>
-        <span className={styles.dialCode}>Int. Dial Code: {dialCode}</span>
-        <span className={styles.language}>Language: {language}</span>
+        <span className={styles.infoStyle}>Country code:{countryCode}</span>
+        <span className={styles.infoStyle}>Currency: {currency}</span>
+        <span className={styles.infoStyle}>Int. Dial Code: {dialCode}</span>
+        <span className={styles.infoStyle}>Language: {language}</span>
+        <span className={styles.infoStyle}>Population: {population}</span>
+        <span className={styles.infoStyle}>Area: {area || 0}</span>
+        <span className={styles.infoStyle}>Borders: {borders || 0}</span>
```
Comment (falecci) [inline src/Components/Countries.jsx:31]: Since `borders` is a string, shouldn't the default value be something like `NONE` or `-` instead of 0?

**Review by falecci — CHANGES_REQUESTED** [review]

File: src/Components/Countries.jsx (hunk 2 above)
Comment (MissMelisa) [reply src/Components/Countries.jsx:31]: checked it

Comment (MissMelisa) [reply src/Components/Countries.jsx:31]: image

**Review by falecci — APPROVED** [review]

**Pull request merged**